main.go

```diff
@@ -92,6 +93,7 @@ var (
 )
 
 var (
+	logFile              = flag.String("log-file", "", "Log to file, if specified. Default is to log to stderr.")
```

Comment:
Is this mutual exclusive such that you can only choose either or between logFile and stderr? What if someone wants both?

Reply:
Good question. If the logging frameworks support it, we could extend this to allow the flag to be specified multiple times.

Reply:
So this is mutually exclusive now right?

Reply:
Correct